frontend/src/loglist/LogsTable.jsx

```diff
@@ -12,96 +11,90 @@ import {
   ChevronRight,
   ChevronLeft
 } from "@material-ui/icons";
-
 import MaterialTable from "material-table";
 import LogInfo from "../model/LogInfo";
+import StreamLogInfo from "../model/StreamLogInfo";
 
-function getColumnInfo() {
-  return [
-    {
-      title: "ID",
-      field: "id",
-      cellStyle: { width: "10%" }
-    },
-    {
-      title: "Date",
-      field: "dateTime",
-      cellStyle: { width: "15%" }
-    },
-    {
-      title: "Level",
-      field: "level",
-      cellStyle: { width: "10%" }
-    },
-    {
-      title: "Message",
-      field: "message",
-      sorting: false
-    }
-  ];
-}
+export default class LogsTable extends React.Component {
```

Comment:
We can't have `DeviceLogTable` and `LogsTable`, either it's log or it's logs. Pick one and stick to it (I prefer singular - log). Then we keep it consistent for logtable, devicelogtable, logapi, logtablewrapper, etc.

Reply:
I saw a lot that were plural already but I didn't notice that singular was the way to go. I think it was just never consistent. I can fix the ones that are relevant to this PR to be singular but I guess we need to make a future PR to go through all the LogTables one by one.

Reply:
Also LogsTable vs DeviceLogTable was naming done prior to this PR, so I didn't actively try to rename them for this PR, but if you think that since I'm already touching these files in my refactor that I should be renaming them, then let me know.

Reply:
Alright let's mark it down for a future cleanup